src/components/Overview/PlotlyCharts/ForceDirectedGraph.tsx

```diff
@@ -1,18 +1,181 @@
 import React from 'react';
```

Comment:
Rename le folder Charts ? since it's not a plotly chart ?

Reply:
✅